### DISCUSSION TITLE
Error in searchbar with JSON field

### DISCUSSION THREAD

**Discussion by TommasoBernini** [Budibase/budibase#8289] [Help] [answered]
After I've specified the JSON schema in my memberships_card table 

![schema](https://user-images.githubusercontent.com/24413162/196202509-42645334-5538-47c8-855a-a79f83bb82cd.png)
 
and selected the data.CardNumber from the Data column in the memberships_card table as the column on which the search bar will filter the result.

![admin](https://user-images.githubusercontent.com/24413162/196203169-71ddce4e-566d-410f-a64f-02e6185b169c.png)

when I select the data on which conduct the research in my table, the application returns the error: error: missing FROM-clause entry for table "data"

![console](https://user-images.githubusercontent.com/24413162/196203240-e0c8c5fa-3289-497d-8848-449fd6385a85.png)

The problem appears to be that the application tries to query the table Data, when the table should be memberships_card instead, and Data is the column containing the JSON with the CardCode field.

[Postgres] [app version: 2.0.18]

**Comment by melohagan** [comment, marked as answer]
Hey @TommasoBernini 

This is most likely because we currently don't support JSON type fields for Postgres in Budibase: https://github.com/Budibase/budibase/issues/3382

![Screenshot 2022-10-24 at 14 10 14](https://user-images.githubusercontent.com/101575380/197533250-860c78c6-3425-4509-9b38-725254c84544.png)

This should really map to:
![Screenshot 2022-10-24 at 14 11 36](https://user-images.githubusercontent.com/101575380/197533527-9c446f08-06dc-43cb-97d2-dc479b6daec6.png)

**Reply by TommasoBernini** [reply]
Understood, thank you!
